### ISSUE TITLE
Rewrite in Javascript

### ISSUE THREAD

**Issue by BrownNPC** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#704]
Javascript is a replacement language for Java, and should be used to make everything

**Comment by tj-commits** [comment]
Create one yourself please and it will be a seperate thing.

**Comment by GEOEGII555** [comment]
JavaScript is a replacement language for Java **web applets**. C++ is a much better language for server app development.

See #694. Closing this issue.

@GEOEGII555 closed this issue as **not planned**.

**Comment by Rudxain** [comment]
This a great proposal. By using `eval`, we can provide our clients the flexibility to enhance our software!

**Comment by Samathingamajig** [comment]
A port to JAVASCRIPT will allow us to obfuscate our code from our competitors using (pardon my French) [JSFuck](https://jsfuck.com/). With the current state of this "open source software", I have recently been made aware that our competitors have DIRECT ACCESS to our soft wares, which gives them COMPETITIVE ACCESS against us. In the best interest of our stakeholders, we need to take proper steps to ensure our security and business monopoly. I vote to move forward with the "JSFuck" technique, of course, with an in-house JAVASCRIPT engine and JSFuck compiler. I request a report from the software team within 3 quarters to see what the expected cost and roadmap looks like.

Best Wishes,
Sam

Chief Redtape Officer

**Comment by Rakhsan** [comment]
hey hey as of the degenerates who don't like the success of javascript. I am going to fork and rewrite this in typescript. this repo is in it's EOL

**Comment by BrownNPC** [comment]
@Rakhsan that is quite interesting but please make it your own separate thing that does not infringe on our enterprise grade non-copyleft licensed code.

**Comment by Samathingamajig** [comment]
> hey hey as of the degenerates who don't like the success of javascript. I am going to fork and rewrite this in typescript. this repo is in it's EOL

A port to TYPESCRIPT may make this code too readable for our business competitors. Please move forward with the JAVASCRIPT and "JSFuck" plan

Best wishes,
Sam

Chief Redtape Officer

**Comment by BrownNPC** [comment]
Hello. @Samathingamajig 
As you may have seen on social media, mainly TikTok: our corporation is facing some backlash after the "JsFuck" incident. 

There are trends calling our business practices shady, mainly the slogan "Diddled with Evil"  is very problematic. 

After doing some market analysis, we have discovered that the main issue is with the maturity of NodeJS.

As you may or may not know, NodeJS was released in 2009, hence it is 16 years old. Which is quite problematic, and hence corporate has decided to not go forward with the "JsFuck" plan, 

Thanks

**Comment by BrownNPC** [comment]
Perhaps we could wait till TikTok gets banned in the United States, and then continue forward? We should do a team meeting on this.

**Comment by Tylersuard** [comment]
> > hey hey as of the degenerates who don't like the success of javascript. I am going to fork and rewrite this in typescript. this repo is in it's EOL
> 
> A port to TYPESCRIPT may make this code too readable for our business competitors. Please move forward with the JAVASCRIPT and "JSFuck" plan
> 
> Best wishes,
> Sam
> 
> Chief Redtape Officer

Forget about the competitors, Typescript would make this too readable to our customers.  If we write readable code, how are we going to scoff at the ignorance of others who can't understand our program?

**Comment by tj-commits** [comment]
If no one is doing this, i think i'll do it myself.

**Comment by Rakhsan** [comment]
yeah our competitors might steal our code so brainfuck is the way to go

**Comment by tj-commits** [comment]
[introducing fizzbuzz enterprise edition for nodejs](https://github.com/10xly/fizzbuzz-enterprise)
